### ISSUE TITLE
Undeclared variable pulse in jscripts.js

### ISSUE THREAD

**Issue by BlackSkorpio** [notandigg/grimsi-geysir#7]
Since this is only used on the front page, it might be better to have it inline there to avoid that JS error
```
pulse.on('start', function () {
  $('#ss_spinner').addClass('loading')
});
pulse.on('waiting', function (b) {
  b ? $('#ss_spinner').addClass('loading')  : $('#ss_spinner').removeClass('loading')
});
pulse.on('dataloaded', function () {
  $('#ss_spinner').removeClass('loading')
});
```

**Comment by notandigg** [comment]
Yes I was meaning to do that :-) It will be gone in next update, which is coming up soon.
Thanks for reminding me about it :-)

**Comment by notandigg** [comment]
This should be fixed now in version 1.6.0
